src/openforms/js/components/admin/form_design/variables/registration/RegistrationsSummary.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import FAIcon from 'components/admin/FAIcon';
+import {BACKEND_OPTIONS_FORMS} from 'components/admin/form_design/registrations';
+
+const RegistrationSummary = ({name, registrationPath}) => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <span>{name}</span>
+      <span>
+        <FAIcon
+          icon="edit"
+          title={intl.formatMessage({
+            defaultMessage: 'Edit variable registration',
+            description: "'Edit variable registration' icon label",
+          })}
+          extraClassname="fa-lg actions__action"
```

Comment:
the `actions__action` class probably doesn't have any effect because there's no `.actions` parent.

```suggestion
          extraClassname="fa-lg"
```

Reply:
I had to use it or it wouldn't to anything on hover

Reply:
it violates the BEM principle, an element cannot exist without being contained inside its parent block.